Source/WebCore/editing/Editor.cpp

```diff
@@ -600,11 +600,11 @@ bool Editor::deleteWithDirection(SelectionDirection direction, TextGranularity g
             // Implicitly calls revealSelectionAfterEditingOperation().
         }
     } else {
-        TypingCommand::Options options = 0;
+        OptionSet<TypingCommand::Option> options = { };
```

Comment:
Nit - `OptionSet`'s default initializer will set all flags to 0, so this should be slightly cleaner:

```suggestion
        OptionSet<TypingCommand::Option> options;
```